packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -5214,9 +5256,13 @@ type StylesheetResource = {
   state: StylesheetState,
 };
 
-export type BoundaryResources = {
+export type BoundaryState = {
```

Comment:
Not sure if this is the best name. I wanted something analagous to RenderState (akin to request state when combined with ResumableState) but since I can't really generalize this state b/c the Fizz runtime needs to understand the concept of hositables to a degree maybe I should name it `HoistableState` or something